src/coordination/coordinator_instance.cpp

```diff
@@ -383,61 +379,145 @@ auto CoordinatorInstance::ForceResetCluster_() -> ForceResetClusterStateStatus {
   spdlog::trace("Acquired lock in the force reset thread {}", std::this_thread::get_id());
   repl_instances_.clear();
 
-  if (!raft_state_->IsLeader()) {
-    spdlog::trace("Exiting force reset as coordinator is not any more leader!");
-    return ForceResetClusterStateStatus::NOT_LEADER;
-  }
-
-  if (!raft_state_->AppendOpenLock()) {
-    spdlog::trace("Appending log force reset failed, aborting force reset");
-    // Here we want to continue doing force reset or kill coordinator. If we can't append open lock,
-    // then we must be follower, and some other coordinator will continue trying force reset. Otherwise
-    // we are leader  but append lock failed, so we are in wrong state.
-    MG_ASSERT(!raft_state_->IsLeader(), "Coordinator is leader but append open lock failed, encountered wrong state.");
-    return ForceResetClusterStateStatus::FAILED_TO_OPEN_LOCK;
-  }
-
   utils::OnScopeExit const do_force_reset_state_update{[this]() {
```

Comment:
Rename do_force_reset to reconciliation or something since we are not using force_reset term anymore

Reply:
`reconciliation`